chain-abci/src/main.rs

```diff
@@ -177,13 +179,22 @@ pub struct AbciOpt {
 
 /// edp
 #[cfg(all(not(feature = "mock-enclave"), feature = "edp", target_os = "linux"))]
-fn get_enclave_proxy() -> TxValidationApp {
-    launch_tx_validation()
+fn get_enclave_proxy(config: &Config, storage: Arc<dyn KeyValueDB>) -> TxValidationApp {
+    let (app, stream_from_tdbe) = launch_tx_validation(config.remote_attestation.address.clone());
+    let tdbe_app = TdbeApp::new(
+        &config.data_bootstrap,
+        &config.remote_attestation,
+        storage,
+        stream_from_tdbe,
+    )
+    .expect("create tdbe app");
+    tdbe_app.spawn();
```

Comment:
This will block.

Reply:
that's correct -- in the current version, it's desired

Reply:
Then maybe change the function name to `run`? `spawn` is a bit misleading.

Reply:
put spawn back

Reply:
+ join / explanation